frontend/actions/review.ts

```diff
@@ -29,16 +29,16 @@ export function useGetReviewRequestsForUser(isActive = true) {
 
 type SemverOrAccessRequestId =
   | {
-      semver: ReleaseInterface['semver']
+      semver: ReleaseInterface['semver'] | ''
```

Comment:
Really sorry, but I've realised the two additions of `|| ''` in this file aren't needed as semver and id are already strings. It's the change we made in another file that allowed us to get rid of the `|| undefined`.